packages/config/src/tokens.ts

```diff
@@ -514,6 +521,13 @@ export const tokenList: TokenInfo[] = [
     decimals: 18,
     sinceBlock: 4719568,
   },
+  {
+    name: 'Wrapped UST Token',
+    symbol: 'UST',
+    address: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
```

Comment:
This is the address of WETH. The correct address is: `0xa47c8bf37f92abed4a126bda807a7b7498661acd`.
Also the activity seems really minimal and this token only has some weird pairs on Uniswap, so getting the price would be problematic. I propose to skip this one for now.
https://etherscan.io/token/0xa47c8bf37f92abed4a126bda807a7b7498661acd?a=0xdc1664458d2f0B6090bEa60A8793A4E66c2F1c00